integration/create_topic_test.go

```diff
@@ -0,0 +1,75 @@
+package integration_test
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/satori/go.uuid"
```

Comment:
could we reuse the UUID lib that we have already in vendor?
`github.com/comail/go-uuid/uuid`

it's a mirror of the old UUID lib in google code